content/docs/conditional-rendering.md

```diff
@@ -36,43 +35,42 @@ function Greeting(props) {
 }
 
 ReactDOM.render(
-  // Try changing to isLoggedIn={true}:
+  // Kodu değiştirerek deneyin: isLoggedIn={true}:
   <Greeting isLoggedIn={false} />,
   document.getElementById('root')
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
+[**Codepen'de deneyin**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+Bu örnek uygulama, `isLoggedIn` değerine göre farklı bir karşılama yapacak.
 
-### Element Variables {#element-variables}
+### Eleman Değişkenleri {#element-variables}
 
-You can use variables to store elements. This can help you conditionally render a part of the component while the rest of the output doesn't change.
+HTML elemanlarını saklamak için değişkenleri kullanabilirsiniz. Bu size, bileşen yaratırken, bileşenin bir bölümünü koşullu hale getimenize yardım eder. 
```

Comment:
`... koşullu hale getimenize ....`

it should be `getirmenize`. you missed the `r` here . Just a small typo.